pages/stable-version.html

```diff
@@ -269,11 +269,13 @@ <h4>Client Tools</h4>
                 <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL8-Uyuni-Client-Tools/EL_8/">Rocky Linux 8 (x86_64, aarch64)</a></li>
                 <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL9-Uyuni-Client-Tools/EL_9/">RHEL, Oracle, AlmaLinux and Rocky Linux 9 (x86_64, ppc64le, aarch64, s390x)</a></li>
                 <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL9-Uyuni-Client-Tools/EL_9/">Oracle Linux (x86_64, ppc64le, aarch64)</a></li>
+                <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/EL9-Uyuni-Client-Tools/EL_9/">Amazon Linux 2023 (x86_64, aarch64)</a></li>
```

Comment:
I just noticed... you probably copied and pasted this, so it's not right
```suggestion
                <li><a href="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL9-Uyuni-Client-Tools/EL_9/">Amazon Linux 2023 (x86_64, aarch64)</a></li>
```